CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.16)

```

Comment:
Is there a need to use CMake here, for just one target? If it's just one command, it can probably be placed inside the Dockerfile directly. Or are you planning on adding more targets?